development/webpack/test/plugins.ManifestPlugin.test.ts

```diff
@@ -281,4 +281,62 @@ describe('ManifestPlugin', () => {
       }
     }
   });
+
+  describe('manifest flags in development mode', () => {
+    const testManifest = {} as chrome.runtime.Manifest;
+    const mockFlags = { remoteFeatureFlags: { testFlag: true } };
+
+    it('adds manifest flags in development mode', () => {
+      const transform = transformManifest(
+        { lockdown: true, test: false },
+        true,
+      );
+      assert(transform, 'transform should be truthy');
+
+      // Mock fs.readFileSync
+      const fs = require('fs');
+      const originalReadFileSync = fs.readFileSync;
+      fs.readFileSync = () => JSON.stringify(mockFlags);
+
+      try {
+        const transformed = transform(testManifest, 'chrome');
+        assert.deepStrictEqual(
+          transformed._flags,
+          mockFlags,
+          'manifest should have flags in development mode',
+        );
+      } finally {
+        // Restore original readFileSync
+        fs.readFileSync = originalReadFileSync;
+      }
+    });
+
+    it('handles missing manifest flags file', () => {
```

Comment:
love that you are testing the error conditions! can you also add a test for the case where the `fs.readFileSync` `error` is _NOT_ `ENOENT`?

Reply:
Sure ! Added `silently ignores non-ENOENT filesystem errors` tests  7c3ff233c0ac3325e0354c5c43bdc1aba427a67a